### PR TITLE
Add dishes

### PR DESCRIPTION
**Pull request by rikikonikoff**

### CODE REVIEW

File: spec/factories/dishes.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :dish do
+    sequence(:creator_id) { |n| }
```
Comment (jennceng) [inline spec/factories/dishes.rb]: what if there are no creators in the database?

Comment (rikikonikoff) [reply spec/factories/dishes.rb]: You have to be signed in to create a dish, and a creator is just an alias for a user! Hence, creator is the current user when a dish is created

File: spec/factories/dishes.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    sequence(:name) { |n| "Al's Pancake World fries #{n}" }
+    description "This is a test dish"
```
Comment (jennceng) [inline spec/factories/dishes.rb:5]: instead I would add user here to hook up your user factory to your dish factory

File: spec/factories/dishes.rb (hunk 1 above)
Comment (jennceng) [reply spec/factories/dishes.rb]: true, but for creating things for test set up (for example viewing a dish show page) you should have a creator in the test database so you can test things like seeing dish.creator

**Pull request merged**